pkg/daemon/constants/constants.go

```diff
@@ -14,6 +14,8 @@ const (
 	DesiredMachineConfigAnnotationKey = "machineconfiguration.openshift.io/desiredConfig"
 	// MachineConfigDaemonStateAnnotationKey is used to fetch the state of the daemon on the machine.
 	MachineConfigDaemonStateAnnotationKey = "machineconfiguration.openshift.io/state"
+	// NodeOnHoldAnnotationKey is used by external components to put a node on hold from being accounted for in the node controller
+	NodeOnHoldAnnotationKey = "machineconfiguration.openshift.io/hold"
```

Comment:
I was thinking about this in the context of reboot requests...how about "prefix annotations", the API is to e.g. add
`machineconfiguration.openshift.io/hold-ceph = "Hold placed by ceph operator"` and we scan for all annotations with that prefix?

This makes them more self-describing and auditable.

Reply:
oh yeah - I went that road but I've been buried into kubernetes specs around annotation and labels that I didn't want to _break_ the way labels/annotations are specified.  I was thinking about `hold.machineconfiguration.openshift.io/<component>` prefixto have an easy split at the slash also. I don't know, I don't feel strongly about this and since, at least for the next release, this is just used by one component we can revisit for 4.3 (we can also mark this beta for 4.2)